### DISCUSSION TITLE
translation to pl

### DISCUSSION THREAD

**Discussion by meavric** [rappasoft/laravel-livewire-tables#1906] [General]
Hi
I see that no all sentenses are translete to polish. How can I translate it?
For example when no data to show in table I see: No items found. Try to broaden your search, but I want to see: Brak danych do wyświetlenia.
I know that i can it by configure      $this->setEmptyMessage('Brak danych do wyświetlenia'); but I want set this comment global in all tables.
BR

**Comment by lrljoe** [comment]
Please either fork the repo, create a new branch, update the pl translation, and then issue a PR against the development branch.

OR as it's a translation, share an updated copy of the whole file here.

**Comment by meavric** [comment]
Hi,
before global public translation I want to check it in my project that all is correct. 
Where put file with translations (pl.json) ?
BR

**Reply by lrljoe** [reply]
It's best to fork the repo, and do your tests in your fork, however you can do:

```
php artisan vendor:publish --tag="livewire-tables-translations"
```

This will publish the translations to:
```
./resources/lang/vendor/livewire-tables/
```

You can also directly modify your local copy of the package.  NOTING - that this will be removed if you update the tables package.
```
vendor/rappasoft/laravel-livewire-tables/resources/lang/pl.json
```

So feel free to share what you have, and I'll include it in the next release.

It will be validated prior to a release anyway.

**Comment by meavric** [comment]
Hi
In attach pl language
BR
[pl.json](https://github.com/user-attachments/files/16835618/pl.json)

**Reply by lrljoe** [reply]
Thanks, I've opened a PR #1925, so this will be in the next release.

Given that it's just a localisation file, the tests will pass, and it will get merged into the "development" branch sometime today, then will get released most likely this weekend.

I have added a credit for you in the localisation section in the contributors file, if you'd prefer that I don't, then let me know before the weekend.

**Comment by meavric** [comment]
OK,
if you add some new phrases i be translate it too.
